pkg/paho-mqtt/patches/0001-MQTTClient-C-skip-SUCCESS-enum-on-STM32L1-L4-WB-cpus.patch

```diff
@@ -0,0 +1,30 @@
+From cbb94538de6b28513685779f542587129f888db6 Mon Sep 17 00:00:00 2001
+From: Alexandre Abadie <alexandre.abadie@inria.fr>
+Date: Sun, 7 Jun 2020 10:27:56 +0200
+Subject: [PATCH 1/1] MQTTClient-C: skip SUCCESS enum on STM32L1/L4/WB cpus
+
+---
+ MQTTClient-C/src/MQTTClient.h | 6 ++++++
+ 1 file changed, 6 insertions(+)
+
+diff --git a/MQTTClient-C/src/MQTTClient.h b/MQTTClient-C/src/MQTTClient.h
+index b612341..a051199 100755
+--- a/MQTTClient-C/src/MQTTClient.h
++++ b/MQTTClient-C/src/MQTTClient.h
+@@ -54,7 +54,13 @@
+ enum QoS { QOS0, QOS1, QOS2, SUBFAIL=0x80 };
+ 
+ /* all failure return codes must be negative */
++#if defined(CPU_FAM_STM32L4) || defined(CPU_FAM_STM32L1) || defined(CPU_FAM_STM32WB)
```

Comment:
Now you update the patch and add `CPU_FAM_STM32G4` to this list.

Reply:
ok will do but please...
#KeepYourManners

Reply:
Yep, sorry, I think one word was missing in my comment:

"Now you **can** update the patch and add CPU_FAM_STM32G4 to this list."

Reply:
ohh got it, it' s ok ;)